.editorconfig

```diff
@@ -0,0 +1,18 @@
+root = true
+
+
```

Comment:
remove this empty line so we only have one between the code blocks

Reply:
Done